TWLight/resources/models.py

```diff
@@ -432,4 +432,52 @@ class Meta:
 
     def __unicode__(self):
         # As with Stream, do not return the partner name here.
-        return self.full_name
\ No newline at end of file
+        return self.full_name
+
+
+
+class Suggest(models.Model):
+
+    class Meta:
+        app_label = 'resources'
+        verbose_name = 'suggestion'
+        verbose_name_plural = 'suggestions'
+        ordering = ['suggested_company_name']
+        
+    suggested_company_name = models.CharField(max_length=40,
+        # Translators: In the administrator interface, this text is help text for a field where staff can add partner suggestions.
+        help_text=_("Potential partner's name (e.g. McFarland)."))
+
+    description = models.TextField(blank=True, max_length=1000,
+        # Translators: In the administrator interface, this text is help text for a field where staff can provide a description of a potential partner.
+        help_text=_("Optional description of this potential partner."))
+
+    company_url = models.URLField(blank=True, null=True,
```

Comment:
let's use the url validator here.
https://docs.djangoproject.com/en/1.11/ref/validators/#urlvalidator

Reply:
sorry, but I'm a bit lost. I thought the point of using a `URLField` was to not have to use `urlvalidator`

Reply:
uh, yep, you can disregard that. My brain made that a charfield for no good reason.